Prep for v1.0.0 release (#325)
bottlerocket-os/bottlerocket-update-operator: CHANGELOG.md

# 1.0.0

## General

Added

> * Mechadism to allow updates in certain update time windows ([#241])

Reviewer: ```suggestion
* Mechanism to constrain updates to a certain time window ([#241])
```

* Option to exclude node before draining - ([#231])
* Port configuration ([#315])
* Support for concurrent updates - ([#238])

> * Automatic prometheus scrapping annotations for controller's service ([#269])

Reviewer: ```suggestion
* Automatic prometheus scraping annotations for controller's service ([#269])
```
